R/parsed.R

```diff
@@ -89,7 +89,8 @@ create_filler <- function(pd_flat) {
       newlines = ~line3 - line2,
       lag_newlines = ~lag(newlines, default = 0),
       col2_nl = ~if_else(newlines > 0L, 0L, col2),
-      spaces = ~col3 - col2_nl - 1L
+      spaces = ~col3 - col2_nl - 1L,
+      multi_line = FALSE
```

Comment:
Not sure. Terminals are not multi-line and we use a post-visitor. If we set it to `NA`, we have to set it to `FALSE` for terminals in order for it to work. Do you think it is worth it in terms of performance?
Also, create filler could be called on the flat structure, which may improve performance. However, then it would not be a transformer anymore and initialising variables would happen before. I think it's worth doing that. What do you think?